pkg/network/setup/network.go

```diff
@@ -67,6 +71,10 @@ func (v VMNetworkConfigurator) getPhase2NICs(domain *api.Domain) ([]podNIC, erro
 		if err != nil {
 			return nil, err
 		}
+		// SR-IOV devices are not part of the phases.
+		if nic.vmiSpecIface.SRIOV != nil {
```

Comment:
if they're removed on the 1st phase, why do we want to check them on the 2nd as well?

Reply:
The SR-IOV are not handled by any phase (1 or 2).
So this change just assures that the objects which refer to SR-IOV are not returned, which in turns assure they will not be processed by any phase or step.

Reply:
I see. ACK